resources/js/common/utils/buildUserAvatarUrl.ts

```diff
@@ -0,0 +1,5 @@
+export function buildUserAvatarUrl(path: string) {
+  const baseUrl = globalThis.mediaUrl;
```

Comment:
this seems to be `undefined` for me:
<img width="1231" height="211" alt="image" src="https://github.com/user-attachments/assets/11b8801a-7936-4aa8-942e-d65ee648b953" />
<img width="328" height="83" alt="image" src="https://github.com/user-attachments/assets/4648c68d-420c-416a-a5d8-b2ee18c26dca" />
<img width="620" height="283" alt="image" src="https://github.com/user-attachments/assets/2774c0db-5550-417d-b3da-cbd26e159611" />

I am a bit confused how it's falling back to the server icon.